src/components/Map/Map.jsx

```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from 'react';
 import { MapContainer, TileLayer, useMap, ZoomControl, Tooltip, Marker } from 'react-leaflet';
```

Comment:
This occurred because the cluster of 18 had a box that was of a different zipcode than the other 17 boxes. Basically, if you have pins all at the same coords but their zip_codes aren't all the same, The point's won't show. I fixed this bug by moving the pin w a different zipcode to a different location.